src/CalcManager/Ratpack/exp.cpp

```diff
@@ -297,7 +297,7 @@ void powrat(PRAT *px, PRAT y, uint32_t radix, int32_t precision)
     }
 }
 
-void powratNumeratorDenominator(PRAT *px, PRAT y, uint32_t radix, int32_t precision)
+void powratNumeratorDenominatorPartial(PRAT *px, PRAT y, uint32_t radix, int32_t precision)
```

Comment:
Since this is using the Power of Powers rule, why don't we name it accordingly, e.g. `powratPowerOfPowers`?

Reply:
I renamed as you suggested.